EfEnumToLookup/LookupGenerator/SqlServerHandler.cs

```diff
@@ -111,12 +117,12 @@ private string PopulateLookup(LookupData lookup)
 	WHEN MATCHED AND src.Name <> dst.Name THEN
 		UPDATE SET Name = src.Name
 	WHEN NOT MATCHED THEN
-		INSERT (Id, Name)
-		VALUES (src.Id, src.Name)
+		INSERT (Id, Name, Description)
```

Comment:
this won't pick up changes in description due to the "when matched" clause above